src/main/java/com/reservation/studio/domain/CancellationPolicy.java

```diff
@@ -0,0 +1,15 @@
+package com.reservation.studio.domain;
+
+import com.reservation.studio.domain.exception.InvalidCancellationThresholdException;
+
+import java.time.Duration;
+import java.util.Objects;
+
+record CancellationPolicy(Duration duration) {
```

Comment:
IMHO trochę mało opisowa nazwa klasy 😁 
To jest `polityka anulowania`, ale w sumie czego?  Domyślam się, że rezerwacji, ale z nazwy to nie wynika.

```suggestion
record ReservationCancellationPolicy(Duration duration) {
```